projectpage/css/compressed/base.min.css

```diff
@@ -0,0 +1 @@
k-large-1-2>.column{-ms-flex:0 0 50%;-webkit-box-flex:0;flex:0 0 50%;max-width:50%}.block-large-full>.column{-ms-flex:0 0 100%;-webkit-box-flex:0;flex:0 0 100%;max-width:100%}@media screen and (max-width: 1200px){.block-medium-1-8>.column{-ms-flex:0 0 12.5%;-webkit-box-flex:0;flex:0 0 12.5%;max-width:12.5%}.block-medium-1-6>.column{-ms-flex:0 0 16.66667%;-webkit-box-flex:0;flex:0 0 16.66667%;max-width:16.66667%}.block-medium-1-5>.column{-ms-flex:0 0 20%;-webkit-box-flex:0;flex:0 0 20%;max-width:20%}.block-medium-1-4>.column{-ms-flex:0 0 25%;-webkit-box-flex:0;flex:0 0 25%;max-width:25%}.block-medium-1-3>.column{-ms-flex:0 0 33.33333%;-webkit-box-flex:0;flex:0 0 33.33333%;max-width:33.33333%}.block-medium-1-2>.column{-ms-flex:0 0 50%;-webkit-box-flex:0;flex:0 0 50%;max-width:50%}.block-medium-full>.column{-ms-flex:0 0 100%;-webkit-box-flex:0;flex:0 0 100%;max-width:100%}}@media screen and (max-width: 800px){.block-tab-1-8>.column{-ms-flex:0 0 12.5%;-webkit-box-flex:0;flex:0 0 12.5%;max-width:12.5%}.block-tab-1-6>.column{-ms-flex:0 0 16.66667%;-webkit-box-flex:0;flex:0 0 16.66667%;max-width:16.66667%}.block-tab-1-5>.column{-ms-flex:0 0 20%;-webkit-box-flex:0;flex:0 0 20%;max-width:20%}.block-tab-1-4>.column{-ms-flex:0 0 25%;-webkit-box-flex:0;flex:0 0 25%;max-width:25%}.block-tab-1-3>.column{-ms-flex:0 0 33.33333%;-webkit-box-flex:0;flex:0 0 33.33333%;max-width:33.33333%}.block-tab-1-2>.column{-ms-flex:0 0 50%;-webkit-box-flex:0;flex:0 0 50%;max-width:50%}.block-tab-full>.column{-ms-flex:0 0 100%;-webkit-box-flex:0;flex:0 0 100%;max-width:100%}}@media screen and (max-width: 600px){.block-mob-1-8>.column{-ms-flex:0 0 12.5%;-webkit-box-flex:0;flex:0 0 12.5%;max-width:12.5%}.block-mob-1-6>.column{-ms-flex:0 0 16.66667%;-webkit-box-flex:0;flex:0 0 16.66667%;max-width:16.66667%}.block-mob-1-5>.column{-ms-flex:0 0 20%;-webkit-box-flex:0;flex:0 0 20%;max-width:20%}.block-mob-1-4>.column{-ms-flex:0 0 25%;-webkit-box-flex:0;flex:0 0 25%;max-width:25%}.block-mob-1-3>.column{-ms-flex:0 0 33.33333%;-webkit-box-flex:0;flex:0 0 33.33333%;max-width:33.33333%}.block-mob-1-2>.column{-ms-flex:0 0 50%;-webkit-box-flex:0;flex:0 0 50%;max-width:50%}.block-mob-full>.column{-ms-flex:0 0 100%;-webkit-box-flex:0;flex:0 0 100%;max-width:100%}}@media screen and (max-width: 400px){.stack>.column{-ms-flex:0 0 100%;-webkit-box-flex:0;flex:0 0 100%;max-width:100%;width:100%;margin-left:0;margin-right:0;padding:0}}.h-group:after{content:"";display:table;clear:both}.is-hidden{display:none}.is-invisible{visibility:hidden}.h-antialiased{-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.h-overflow-hidden{overflow:hidden}.h-remove-top{margin-top:0}.h-remove-bottom{margin-bottom:0}.h-add-half-bottom{margin-bottom:1.6rem !important}.h-add-bottom{margin-bottom:3.2rem !important}.h-no-border{border:none}.h-full-width{width:100%}.h-text-center{text-align:center}.h-text-left{text-align:left}.h-text-right{text-align:right}.h-pull-left{float:left}.h-pull-right{float:right}
```

Comment:
are minified css generated during build? if so, I am not sure if it should be pushed.

Reply:
@sakshigupta265 Can you check this since it's part of the project page?